index.js

```diff
@@ -16,6 +16,7 @@ const authors = {
   hueniverse: 'https://www.patreon.com/eranhammer',
   hughsk: 'https://hughsk.io/donate/',
   jayphelps: 'https://www.patreon.com/jayphelps',
+  '~johnjleider': 'https://www.patreon.com/vuetify',
```

Comment:
Lol, sorry this is still wrong. The username doesn't have a `~` in it. You can see how all the other usernames are done right next to yours. I'll just fix this and merge.